src/app/clusters/scenes/scenes.cpp

```diff
@@ -718,23 +718,22 @@ EmberAfStatus emberAfScenesClusterRecallSavedSceneCallback(EndpointId endpoint,
 }
 
 bool emberAfPluginScenesServerParseAddScene(app::CommandHandler * commandObj, const EmberAfClusterCommand * cmd, GroupId groupId,
-                                            uint8_t sceneId, uint16_t transitionTime, uint8_t * sceneName,
+                                            uint8_t sceneId, uint16_t transitionTime, chip::ByteSpan sceneName,
                                             uint8_t * extensionFieldSets)
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
     EmberAfSceneTableEntry entry;
     EmberAfStatus status;
     bool enhanced                  = (cmd->commandId == ZCL_ENHANCED_ADD_SCENE_COMMAND_ID);
     uint16_t extensionFieldSetsLen = static_cast<uint16_t>(
-        cmd->bufLen -
-        (cmd->payloadStartIndex + sizeof(groupId) + sizeof(sceneId) + sizeof(transitionTime) + emberAfStringLength(sceneName) + 1));
+        cmd->bufLen - (cmd->payloadStartIndex + sizeof(groupId) + sizeof(sceneId) + sizeof(transitionTime) + sceneName.size() + 1));
```

Comment:
This all does not make sense with TLV encoding.  Please file a followup issue on this?